src/Interro.java

```diff
@@ -4,26 +4,41 @@
 import java.io.FileWriter; import java.io.PrintWriter; import java.io.File; import java.io.FileNotFoundException; import java.io.IOException;
 public class Interro {
     int width, height;
+    CardLayout cL;
     JFrame frame;
     ArrayList<Game> privateGames = new ArrayList();
     ArrayList<Game> publicGames = new ArrayList();
     ArrayList<Team> teams = new ArrayList();
-    Home homePage;
-    TeamPage teamPage;
+    Panel panel;
+    Game game; Home homePage; TeamPage teamPage;
     public static void main(String[] args){new Interro();}
     public Interro(){
-        width = 1000; height = 1000;
+        width = 1010; height = 1000;
 
-        homePage = new Home();
-        teamPage = new TeamPage();
+        cL = new CardLayout();
+
+        panel = new Panel();
 
         frame = new JFrame("Test");
-        frame.setSize(width, height);
+        frame.setSize(width, height+frame.getInsets().top);
         frame.setResizable(true);
         frame.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
-        frame.setContentPane(homePage);
+        frame.setContentPane(panel);
         frame.setVisible(true);
     }
+    class Panel extends JPanel{
```

Comment:
Skip a line between class definitions.

Also, since this is an inner class, make it `static` if possible or, if it makes sense, extract it to a new file.